ledger/sandbox-classic/src/test/suite/scala/platform/sandbox/services/reset/ResetServiceH2DatabaseIT.scala

```diff
@@ -4,11 +4,16 @@
 package com.daml.platform.sandbox.services.reset
 
 import com.daml.platform.sandbox.SandboxBackend
-import com.daml.platform.sandbox.services.SandboxFixture
+import com.daml.platform.sandbox.services.{SandboxEnableAppendOnlySchema, SandboxFixture}
 
 final class ResetServiceH2DatabaseIT
     extends ResetServiceDatabaseIT
     with SandboxFixture
     with SandboxBackend.H2Database
 
-// TODO append-only: add a test for H2 on the append-only schema
+// TODO append-only: remove this class once the append-only schema is the default one
+final class ResetServiceH2AppendOnlyIT
```

Comment:
@rautenrieth-da  can you please verify that this is the only place where I should have added coverage with H2+Append-only?